src/pages/landing-page/landingPage.jsx

```diff
@@ -1,7 +1,8 @@
 import "./landingPage.css";
 
 import {useCategory} from "../../context/category.context";
-<script defer src="https://friconix.com/cdn/friconix-0.2248.js"> </script>
+import { FiArrowRight } from "react-icons/fi";
+import { FiChevronRight } from "react-icons/fi";
 const LandingPage = () =>
```

Comment:
as you are importing from same file , you can make write it in a single line, as I told before